specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/stable/2023-07-20/examples/AzureLargeInstance_Start.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "api-version": "2023-07-20",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "azureLargeInstanceName": "myALInstance"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.AppPlatform/locations/eastus/operationStatus/default/operationId/00000000-0000-0000-0000-000000000000?api-version=2023-07-20"
```

Comment:
The response of our 200 is the iformation of the operationStatus. Please update it.